packages/pages/src/vite-plugin/build/build.ts

```diff
@@ -60,22 +57,11 @@ export const build = (projectStructure: ProjectStructure): Plugin => {
             },
           },
           reportCompressedSize: false,
+          assetsDir: projectStructure.config.subfolders.assets,
         },
-        define: processEnvVariables(projectStructure.envVarPrefix),
-        base: "", // makes static assets relative for reverse proxies and doesn't affect non-RP sites
```

Comment:
I think it's probably worth detailing how someone would migrate their assets to the new way if they've already updated to this case.

A migration to 1.0 guide will be very useful.